src/display/editor/editor.js

```diff
@@ -457,10 +461,16 @@ class AnnotationEditor {
       boundResizerPointermove,
       pointerMoveOptions
     );
+    const savedX = this.x;
+    const savedY = this.y;
+    const savedWidth = this.width;
+    const savedHeight = this.height;
+    const savedParentCursor = this.parent.div.style.cursor;
+    const savedCursor = this.div.style.cursor;
+    this.div.style.cursor = this.parent.div.style.cursor =
+      window.getComputedStyle(event.target).cursor;
+
     const pointerUpCallback = () => {
-      // Stop the undo accumulation in order to have an undo action for each
-      // resize session.
-      this._uiManager.stopUndoAccumulation();
       this._isDraggable = savedDraggable;
       this.parent.div.classList.remove(resizingClassName);
```

Comment:
Looking at the updated CSS, do we still use the `resizingClassName` for anything?

If not this line, together with https://github.com/mozilla/pdf.js/blob/915352ec2228be5d836b7a405c633112a0bbe57a/src/display/editor/editor.js#L454-L457, can probably be removed now.